src/models.py

```diff
@@ -348,9 +362,10 @@ def build_embeddings(args, vocab, tasks, pretrained_embs=None):
     assert d_emb, "You turned off all the embeddings, ya goof!"
     return d_emb, embedder, cove_layer
 
-
-def build_module(task, model, d_sent, d_emb, vocab, embedder, args):
-    ''' Build task-specific components for a task and add them to model. '''
+def build_task_specific_components(task, model, d_sent, d_emb, vocab, embedder, args):
```

Comment:
nit: could you rename this to `_build_task_module`? The name is really close to `build_task_modules` as is, and the underscore I think will make a little clearer what's going on.